css/main.css

```diff
@@ -0,0 +1,75 @@
+* {
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 700;
+}
+
+::-webkit-scrollbar {
+  width: 6px;
+  height: 10px;
+  display: none;
+}
+
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.logo > img {
+  width: 80px;
+  height: 80px;
+  padding: 10px;
+}
+
+header > .button {
+  padding: 30px;
+}
+
+.hero-section {
+  background-image: url("../image/heroImage.jpg");
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
```

Comment:
- [ ] Kyndly adjust the background image to look as showed in the figma design, you made an excelent job using the css background-image, but it seems you are not usig the correct background.

Your design

![image](https://user-images.githubusercontent.com/95243996/191733538-336397f0-267c-4d08-ac6f-e3d024a52289.png)

 Figma

![image](https://user-images.githubusercontent.com/95243996/191733853-558f16f3-9362-4255-b240-09e80e66e54c.png)

- If you click on the header-shapes-mobile on figma, on the right side you will see the option to export it.

![image](https://user-images.githubusercontent.com/95243996/191734388-82a0e9f2-e3b3-4e22-8089-b0f3f34b0bc8.png)

![image](https://user-images.githubusercontent.com/95243996/191734637-a23b416c-4016-47c3-96e5-62f47b8f572b.png)